go/test/endtoend/reparent/plannedreparent/reparent_test.go

```diff
@@ -26,10 +26,13 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"google.golang.org/protobuf/encoding/protojson"
```

Comment:
this should be with the above 3rd-party imports

Reply:
Our coding standard doesn't really say anything about the import order, does it? We only need the linter check to pass, right?
I think that if we are trying to follow a more specific standard than what is imposed by the linter, we should be clear about it because I am not entirely sure what it is. I try to follow the standard so that I don't need another commit to fix them later.

Reply:
you're right, we don't have a single convention (primarily because vitess is really half a dozen codebases all shoved into one repo), but what i've been doing in the vtctldserver code (which the `reparentutil` stuff came out of) is:

```go
import (
    // section 1: stdlib

    // section 2: 3rd-party

    // section2b (almost never): 3rd-party proto imports, renamed from `<pkg>` to `<pkg>pb`

    // section 3: vitess imports

    // section3b: vitess proto imports, renamed with `pb` suffix
)
```

in my opinion, it's clean, it's clear, it's not "randomly where did your code editor autocomplete decide to put this", and you avoid accidentally duplicating imports (especially with the proto renames, see https://github.com/vitessio/vitess/pull/10904#discussion_r934561222 for example)